### DISCUSSION TITLE
cannot recognize entities?

### DISCUSSION THREAD

**Discussion by pchalasani** [explosion/spaCy#8688] [Help: Coding & Implementations] [answered]
Very puzzled by this -- what am I missing?

```
In [1]: import spacy

In [2]: nlp = spacy.load("en_core_web_sm")

In [3]: doc = nlp("Welcome to Google, Victor!")

In [4]: doc.ents
Out[5]: ()
```

**Comment by pchalasani** [comment, marked as answer]
A bit better with `en_core_web_trf`: it got Victor but not Google.

```
In [2]: nlp = spacy.load("en_core_web_trf")

In [3]: doc = nlp("Welcome to Google, Victor!")

In [4]: doc.ents
Out[4]: (Victor,)
```

**Reply by polm** [reply]
Please see #3052. 

It's a little surprising this doesn't work as expected, but note that since the spaCy models are trained on newspaper texts they don't have many examples of dialog-type text, and that the articles used for training data probably predate Google.

**Reply by polm** [reply]
Note that a similar non-dialog sentence, "Victor works at Google.", gets both entities as expected.
